docs/responses/source.rst

```diff
@@ -6,100 +6,47 @@ Give us the source of the status.
 .. note::
     The API equivalent is :code:`status_source`.
 
+
 HTTP Code
 ^^^^^^^^^
 
 This source is returned when **all the following cases** are met:
 
-- We can't extract the expiration date from :func:`PyFunceble.lookup.whois.WhoisLookup.request`.
+- We can't extract the expiration date from
+  :func:`PyFunceble.lookup.whois.WhoisLookup.request`.
```

Comment:
Not your fault but let's update it !

:class:`PyFunceble.query.whois.query_tool.WhoisQueryTool` should be correct now 😄 

In other words (previous is weird): `func` is now `class` and `PyFunceble.lookup.whois.WhoisLookup.request` is now `PyFunceble.query.whois.query_tool.WhoisQueryTool`.

Reply:
Could you throw these into a [todo](https://github.com/spirillen/PyFunceble/issues)?

Reply:
Should have been solved
/move to done https://github.com/funilrys/PyFunceble/projects/7#card-52531230

Reply:
@funilrys is there something which is hanging in this comment?? since it is mention about 5 times in the project? 

- https://github.com/funilrys/PyFunceble/projects/7#card-52531230
- https://github.com/funilrys/PyFunceble/projects/7#card-52531238
- https://github.com/funilrys/PyFunceble/projects/7#card-52531257
- https://github.com/funilrys/PyFunceble/projects/7#card-52531327
- https://github.com/funilrys/PyFunceble/projects/7#card-52531336